pkg/resources/scim_integration_acceptance_test.go

```diff
@@ -164,14 +164,14 @@ func TestAcc_ScimIntegration_invalid(t *testing.T) {
 		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
 			tfversion.RequireAbove(tfversion.Version1_5_0),
 		},
+		ErrorCheck: helpers.AssertErrorContainsPartsFunc(t, []string{
```

Comment:
hmm, the usual way we follow is using ExpectError, not the error check (I think I have already written it in the previous PR)

Reply:
Okay, but I noticed that order of some errors is not deterministic, so it's better to check it here instead of using a regex.

Reply:
So usually it's better to have a configuration for test which will result in just one error; you don't have the order problem then (check warekouse acceptance test)

Reply:
I'll split this into two